integrations/service/node/utils/index.ts

```diff
@@ -0,0 +1,151 @@
+export const getRandomInt = (): number => {
+  const min = 1;
+  const max = 99999;
+  return Math.floor(Math.random() * (max - min) + min); // The maximum is exclusive and the minimum is inclusive
+};
+
+export const generateAccountData = (): Record<string, unknown> => {
```

Comment:
overall the data here should be minimal for demo site with limited field to demonstrate prefill feature